planning/behavior_path_planner/config/behavior_path_planner.param.yaml

```diff
@@ -18,3 +18,9 @@
     intersection_search_distance: 30.0
 
     visualize_drivable_area_for_shared_linestrings_lanelet: true
+
+    lateral_distance_max_threshold: 5.0
```

Comment:
Previously we found situation where the vehicle came a little bit too near to the parked lorry (at Telecom center), so at the moment, this parameter is made to be a little bit conservative.

Reply:
ok. It would be better to think about the actual lane width as well. Not only the distance. What do you think?
If so, please keep it in the readme as a limitation.

Reply:
Yes, considering actual lane width might be a good option, but since it is more of the object's relative lateral distance, I think it might be feasible to just keep it as it is and tune it in the product level.